packages/server-wallet/src/message-service/legacy-engine-message-service.ts

```diff
@@ -0,0 +1,53 @@
+import {Message} from '@statechannels/client-api-schema';
```

Comment:
Should the filename specify that this is test code?

Reply:
Yup, that's a good point!

Reply:
a4ac420